packages/opentelemetry-api/src/metrics/Metric.ts

```diff
@@ -101,7 +101,7 @@ export interface Metric<T> {
 
 export interface MetricUtils {
   /**
-   * Adds the given value to the current value. Values cannot be negative.
+   * Adds the given value to the current value. Values cannot be negative.
```

Comment:
what are these whitespace changes?

Reply:
it was <nbsp> and it's not recommended by eslint...

Reply:
I don't know how to interpret your answer

Reply:
sorry `<>` is hidden in github

Reply:
ref: https://eslint.org/docs/rules/no-irregular-whitespace